Gemfile.lock

```diff
@@ -445,7 +445,6 @@ GEM
       strings (~> 0.2.0)
       tty-screen (~> 0.8)
     unf_ext (0.0.8.2)
-    unf_ext (0.0.8.2-x64-mingw-ucrt)
```

Comment:
Should this be removed? I feel like Windows does that for some reason but we still need it?